randomize/plugin.py

```diff
@@ -19,8 +19,37 @@
 from nose.util import isclass, isgenerator, transplant_func, transplant_class
 import random
 import unittest
+from random import getrandbits
+from nose.tools import nottest
 
 log = logging.getLogger(__name__)
+CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME = 'randomize_tests_seed'
+
+
+@nottest
+def randomize_tests(seed=None):
+    """
+     The randomize_tests decorator
+
+     It supports using a predefined seed, otherwise a random seed will be generated.
+     Usage:
+
+         @randomize_tests()
+         class MyClass(unittest.TestCase):
+            pass
+
+        @randomize_tests(seed=1723930311)
+        class MyOtherClass(unittest.TestCase):
+            pass
+    """
+    def rebuild(cls):
+        setattr(cls, CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME, seed)
+        if seed is None:
+            setattr(cls, CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME, getrandbits(32))
```

Comment:
Per the comment above: this would go from `getrandbits(32)` to `random.getrandbits(32)`